ISPN-12622 Expose Cross-Site Views via CLI and REST endpoint (#9008)
infinispan/infinispan: documentation/src/main/asciidoc/topics/ref_rest_cache_managers.adoc

* `cluster_members` and `cluster_members_physical_addresses` an array of logical and physical addresses of the members of the cluster
* `cluster_size` number of members in the cluster
* `defined_caches` A list of all caches defined in the cache manager, excluding private caches but including internal caches that are accessible

> * `local_site` The local site name.

Reviewer: ```suggestion
* `local_site` The name of the local site. +
```

> If cross-site replication isn't configured, it returns "local".

Reviewer: ```suggestion
If cross-site replication is not configured, {brandname} returns "local".
```

> * `sites_view` The cross-site replication view.

Reviewer: ```suggestion
* `sites_view` The list of sites that participate in cross-site replication. +
```

> If not configured, it returns an empty list.

Reviewer: ```suggestion
If cross-site replication is not configured, {brandname} returns an empty list.
```
